### DISCUSSION TITLE
How to use secure websocket (wss) in rosbridge websocket server?

### DISCUSSION THREAD

**Discussion by feb-ri** [RobotWebTools/rosbridge_suite#759] [Q&A] [answered]
How to implement secure websocket with rosbridge? I tried a workaround on this [thread](https://answers.ros.org/question/198647/rosbridge-websocket-ssl-connection-aborted/), but failed. 

Has anyone succesfully implemented wss protocol?

**Comment by jtbandes** [comment, marked as answer]
Does this answer your question? https://github.com/RobotWebTools/rosbridge_suite/issues/232

**Reply by feb-ri** [reply]
Thanks for the answer. Gotta look into it first.
